### PR TITLE
Dual publish

### PR DESCRIPTION
**Pull request by L2jLiga**
based on #11 

known issues: https://github.com/ai/dual-publish/issues/1

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -50,4 +50,4 @@ var crossTab = function (config) {
  * @param {*} Event data
  */
 
-module.exports = crossTab
+module.exports = { crossTab }
```
Comment (ivan-solovev) [inline index.js:53]: Why was the export changed?

Comment (ivan-solovev) [reply index.js:53]: @ai why switch to named export?

Comment (ai) [reply index.js:53]: Because webpack does not support default ESM export in some cases and also pack named exports much more efficient.

Comment (ivan-solovev) [reply index.js:53]: Ok. Thank you for your explanation.

**Comment by ivan-solovev** [comment]
Thank you for the PR!

**Pull request merged**